test/transactions/3_castVotes.js

```diff
@@ -164,4 +164,76 @@ describe('#castVotes transaction', () => {
 				.and.be.hexString();
 		});
 	});
+
+	describe('when the cast vote transaction is created with the votes and unvotes', () => {
+		beforeEach(() => {
+			castVotesTransaction = castVotes({
+				passphrase,
+				votes: votePublicKeys,
+				unvotes: unvotePublicKeys,
+			});
+		});
+
+		it('the transaction should have the votes object', () => {
```

Comment:
Not sure this is necessary given the next test, but if it is, isn't it an array?